internal/lambdalabs/v1/client.go

```diff
@@ -12,15 +14,21 @@ type LambdaLabsClient struct {
 	v1.NotImplCloudClient
 	apiKey  string
 	baseURL string
+	client  *openapi.APIClient
 }
 
 var _ v1.CloudClient = &LambdaLabsClient{}
 
 // NewLambdaLabsClient creates a new Lambda Labs client
 func NewLambdaLabsClient(apiKey string) *LambdaLabsClient {
```

Comment:
lets implement a cloudcred that turns into a client. This cloudcred should hold a RefId